test/gtest/common/test_ucc.cc

```diff
@@ -391,13 +399,60 @@ void proc_context_create(UccProcess_h proc, int id, ThreadAllgather *ta, bool is
     throw std::runtime_error(err_msg.str());
 }
 
+void proc_onesided_context_create(UccProcess_h proc, int id,
```

Comment:
the name "proc_onesided_context_create" is misleading imho. I would rather called it proc_context_create_mem_params

Reply:
done